doc/classes/Image.xml

```diff
@@ -430,8 +430,10 @@
 			</return>
 			<argument index="0" name="square" type="bool" default="false">
 			</argument>
+			<argument index="1" name="interpolation" type="int" enum="Image.Interpolation" default="1">
+			</argument>
 			<description>
-				Resizes the image to the nearest power of 2 for the width and height. If [code]square[/code] is [code]true[/code] then set width and height to be the same.
+				Resizes the image to the nearest power of 2 for the width and height. If [code]square[/code] is [code]true[/code] then set width and height to be the same. New pixels are calculated using [code]interpolation[/code]. See [code]interpolation[/code] constants.
```

Comment:
```suggestion
				Resizes the image to the nearest power of 2 for the width and height. If [code]square[/code] is [code]true[/code] then set width and height to be the same. New pixels are calculated using the [code]interpolation[/code] mode defined via [enum Interpolation] constants.
```

Reply:
I just copied that bit from the `resize()` documentation above, so we should change it as well?

https://github.com/godotengine/godot/blob/053254466115eae12e48462c0c503280a21f1544/doc/classes/Image.xml#L415-L426

Reply:
Yeah that would be good. The part about `[code]interpolation[/code] constants` is incorrect, that's not a proper link to the enum.

Reply:
Done, updated documentation of both methods.